static/app/components/replays/diff/replayDiffChooser.tsx

```diff
@@ -53,6 +56,18 @@ export default function ReplayDiffChooser({
             />
           </TabPanels.Item>
           <TabPanels.Item key={DiffType.HTML}>
+            <StyledAlert type="info" showIcon>
+              {tct(
+                `This feature may be buggy, so we are looking at other potential solutions. Weigh in on these solutions [link:here]`,
+                {
+                  link: (
+                    <ExternalLink
+                      href={'https://github.com/getsentry/sentry/issues/80074'}
```

Comment:
Will be replacing the link with a new github issue that includes all 3 potential solutions